deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml

```diff
@@ -29,6 +36,33 @@ spec:
       name: clustermanagers.operator.open-cluster-management.io
       version: v1
   displayName: Cluster Manager
+  description: |
+    The Cluster Manager provides the multicluster hub, which can manage Kubernetes-based clusters across data centers, public clouds, and private clouds. This operator supports the installation and upgrade of Cluster Manager.
+
+    ## How to Install
+    Install Cluster Manager operator by following instructions in top right button `Install`. After installing the operator, create an instance of the ClusterManager resource to install the Cluster Mananger.
+
+    The pods of Cluster Manager are created in the `open-cluster-management-hub` namespace:
+
+    ```
+    $ oc get pods -n open-cluster-management-hub
```

Comment:
We should probably use `kubectl` in these instructions in the context of this repo.

Reply:
done